soc/st/stm32/soc.yml

```diff
@@ -101,6 +101,12 @@ family:
     - name: stm32h562xx
     - name: stm32h563xx
     - name: stm32h573xx
+  - name: stm32h7rsx
+    socs:
+    - name: stm32h7r3xx
+    - name: stm32h7r7xx
+    - name: stm32h7s3xx
+    - name: stm32h7s7xx
```

Comment:
Could it be moved to initial commit ?

Reply:
moved in the commit d1715d0cafbd9b9dd7ac1194e7a5c12a421a6ac0